dts/riscv/riscv32-fe310.dtsi

```diff
@@ -93,6 +93,15 @@
 			label = "gpio_0";
 			status = "disabled";
 			#gpio-cells = <2>;
+
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges;
+
+			pinctrl: pinctrl@10012000 {
+				compatible = "sifive,iof";
+				reg = <0x10012038 0x8>;
```

Comment:
Also this is wrong from the start.  `IOF_SEL` is offset 60 bytes, which is 0x3C, from the GPIO start, and it's only 4 bytes long.  So this should be `reg = <0x1001203c 0x04>;`

Reply:
No, I'm wrong, 0x38 is IOF_EN and 0x3C is IOF_SEL.  Distracted by a telcon.

So the only concern is that the unit address doesn't match the register.